src/pages/learn/index.tsx

```diff
@@ -0,0 +1,729 @@
+  const intl = useIntl()
+  const isRightToLeft = isLangRightToLeft(intl.locale as Lang)
+
+  const tocItems = [
+    {
+      id: "what-is-crypto-ethereum",
+      title: "What is crypto and Ethereum?",
+    },
+    {
+      id: "how-do-i-use-ethereum",
+      title: "How do I use Ethereum?",
+    },
+    {
+      id: "what-is-ethereum-used-for",
+      title: "What is Ethereum used for?",
+    },
+    {
+      id: "strengthen-the-ethereum-network",
+      title: "Strengthen the Ethereum network",
+    },
+    {
+      id: "learn-about-ethereum-the-protocol",
+      title: "Learn about Ethereum the protocol",
+    },
+    {
+      id: "learn-about-the-ethereum-community",
+      title: "Learn about the Ethereum community",
+    },
+  ]
+
+  const heroContent = {
+    title: "Learn Hub",
+    header: "Learn about Ethereum",
+    subtitle:
+      "Your educational guide to the world of Ethereum. Learn how Ethereum works and how to connect to it. This page includes technical and non-technical articles, guides, and resources.",
+    image: getImage(data.heroImage),
+    alt: "", // TODO
+    buttons: [
+      {
+        content: "Let's get started",
+        pathId: tocItems[0].id,
+      },
+    ],
+  }
+
+  return (
+    <Container>
+      <PageMetadata title={"Learn Hub"} description={""} />
+
+      <HeroBackground>
+        <HeroContainer>
+          <Hero content={heroContent} isReverse />
+        </HeroContainer>
+      </HeroBackground>
+
+      <Page dir={isRightToLeft ? "rtl" : "ltr"}>
+        <InfoColumn>
+          <InfoTitle>Learn Hub</InfoTitle>
+          <StakingHomeTableOfContents items={tocItems} />
+        </InfoColumn>
+
+        <ContentContainer id="content">
+          <Section>
+            <h2 id={tocItems[0].id}>{tocItems[0].title}</h2>
+            <p>
+              You have probably heard a thing or two about cryptocurrencies,
+              Bitcoin and blockchain, but do you know what those actually are
+              and how they relate to Ethereum? And what is Ethereum anyway?{" "}
+              <strong>
+                Check out our introductory{" "}
+                <Link to="/what-is-ethereum/">What is Ethereum</Link> page.
+              </strong>
+            </p>
+            <p>
+              Not only that Ethereum can do what Bitcoin does (transfer money
+              globally), it’s capable of a lot more – people can actually deploy
+              code onto the network. Because it’s so flexible, any kind of
+              computer application can be launched on Ethereum.
+            </p>
+            <CardGrid>
+              <Card
+                title="What is Ethereum?"
+                description="If you are new, start here and learn the basics of what
+              crypto and Ethereum are and why they matter."
+              >
+                <>
+                  <Center>
+                    <GatsbyImage image={getImage(data.whatIsEth)} />
```

Comment:
I know `GatsbyImage` defaults to `alt=""` but IMO we should get into the habit of being explicit with blank alts on images. Otherwise, it's a huge amount of a11y debt to pay back if we migrate to another framework.

```suggestion
                    <GatsbyImage image={getImage(data.whatIsEth)} alt="" />
```

Reply:
Good point. I'll try to add an alt to all the images.